test/FormCheckSpec.tsx

```diff
@@ -0,0 +1,264 @@
+import { render } from '@testing-library/react';
+import * as React from 'react';
+import FormCheck from '../src/FormCheck';
+import Switch from '../src/Switch';
+
+describe('<FormCheck>', () => {
+  it('should render correctly', () => {
+    const { getByTestId, container } = render(
+      <FormCheck
+        id="foo"
+        name="foo"
+        value="foo"
+        defaultChecked
+        label="My label"
+        className="my-checkbox"
+        data-testid="test-id"
+      />,
+    );
+
+    const element = getByTestId('test-id');
+
+    element.parentElement.classList.length.should.equal(2);
+    element.parentElement.classList.contains('form-check').should.be.true;
+    element.parentElement.classList.contains('my-checkbox').should.be.true;
+
+    element.id.should.equal('foo');
+    element.classList.length.should.equal(1);
+    element.classList.contains('form-check-input').should.be.true;
+    element.hasAttribute('name').should.be.true;
+    element.getAttribute('name').should.equal('foo');
+    element.hasAttribute('type').should.be.true;
+    element.getAttribute('type').should.equal('checkbox');
+    element.hasAttribute('value').should.be.true;
+    element.getAttribute('value').should.equal('foo');
+    element.hasAttribute('checked');
+
+    const labels = container.getElementsByTagName('label');
+    labels.length.should.equal(1);
+    const label = labels[0];
+    label.classList.length.should.equal(1);
+    label.classList.contains('form-check-label').should.be.true;
+    label.hasAttribute('for').should.be.true;
+    label.getAttribute('for').should.equal('foo');
```

Comment:
Don't need hasAttribute since the value is checked on next line anyway

Reply:
Fixed in the commit https://github.com/react-bootstrap/react-bootstrap/pull/6224/commits/023f9b3e2cffbb507001f992d76bb9cfb4243310